tests/src/core/resolve.js

```diff
@@ -99,13 +99,14 @@ describe('resolve', function () {
                       )).to.equal(utils.testFilePath('./bar.jsx'))
   })
 
-  it('finds resolvers from the source files rather than eslint-module-utils', function () {
-    const testContext = utils.testContext({ 'import/resolver': { 'foo': {} } })
-
-    expect(resolve( '../files/foo'
-                      , Object.assign({}, testContext, { getFilename: function () { return utils.getFilename('foo.js') } }),
-                      )).to.equal(utils.testFilePath('./bar.jsx'))
-  })
+  // crash because context.report doesn't exist
```

Comment:
why would `context.report` not exist?

Reply:
have you try to run the tests ? context was always null for me on windows

Reply:
That doesn't make any sense; `context.report` is the way eslint creates all of its warnings.